### DISCUSSION TITLE
Colours for the calculator

### DISCUSSION THREAD

**Discussion by Felix-E-S** [Felix-E-S/T24S2T1EXTG08#4] [General]
Hi,
I was just wondering if we want to have a colour scheme for this project and keep all the different windows (Main Menu, Calculator, Currency Converter and Mortgage Calculator) in the same colours.
What do you think? Do we use specific colours or just stick to how we have it?

**Comment by StutzTAFE** [comment]
I think sticking to what we have now would be fine. I'm going to add some colour to mine so its not just black.
